# Statistical Analysis of High-Throughput Genomic and Transcriptomic Data
Fall/Herbst-semester 2023

## Lectures
Mondays 9.00-9.45 (Y27-H-46), 10.00-10.45 (Y27-H-46)

## Exercises
Monday 11.00-11.45

## Lecturers

[Dr. Hubert Rehrauer](http://www.fgcz.ch/the-center/people/rehrauer.html), Group Leader of Genome Informatics at [FGCZ](http://www.fgcz.ch/)  

[Prof. Dr. Mark Robinson](https://robinsonlabuzh.github.io/), Professor of Statistical Genomics, [DMLS](https://www.mls.uzh.ch/en.html), UZH  


## Schedule

| Date  | Lecturer | Topic | Exercise | JC1 | JC2 |
| --- | --- | --- | --- | --- | --- |
| 18.09.2023  | Mark + Hubert  | admin; mol. bio. basics | quarto; git(hub) | | |
| 25.09.2023  | Mark | interactive technology/statistics session  | group exercise: technology pull request | | |
| 02.10.2023  | Hubert | NGS intro; exploratory data analysis | EDA in R | | |
| 09.10.2023  | Mark | limma + friends | linear model simulation + design matrices | | |
| 16.10.2023  | Hubert | mapping  | Rsubread | | |
| 23.10.2023  | Hubert | RNA-seq quantification | RSEM  |[SEACells infers transcriptional and epigenomic cellular states from single-cell genomics data](https://www.nature.com/articles/s41587-023-01716-9) (MB, HW) | X |
| 30.10.2023  | Mark | edgeR+friends 1 | basic edgeR/voom | [Normalization of RNA-seq data using factor analysis](https://www.nature.com/articles/nbt.2931) (MR, RD)  | X |
| 06.11.2023  | tba | hands-on session #1: RNA-seq | FASTQC/Salmon/etc. | [Statistical significance for genomewide studies](https://www.pnas.org/doi/epdf/10.1073/pnas.1530509100) (DA, KS, FM) | [Identification of cell types, states and programs by learning gene set representations](https://www.biorxiv.org/content/10.1101/2023.09.08.556842v1.full.pdf) (TO, MC, GC) |
| 13.11.2023  | Mark | edgeR+friends 2  | advanced edgeR/voom | [OUTRIDER:A novel hierarchical clustering algorithm for gene sequences](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-174) (AB, PB, CD) | [Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4772-0) (DB, CB) |
| 20.11.2023  | Hubert | single-cell 1: preprocessing, dim. reduction, clustering | clustering | X | X |
| 27.11.2023  | tba | hands-on session #2: cytometry | cytof null comparison | [SpatialDM for rapid identification of spatially co-expressed ligand–receptor and revealing cell–cell communication patterns](https://www.nature.com/articles/s41467-023-39608-w) (EG, AE) | [Differential abundance testing on single-cell data using k-nearest neighbor graphs](https://www.nature.com/articles/s41587-021-01033-z)(CC, ZY, XY) |
| 04.12.2023  | Mark | single-cell 2: clustering, marker gene DE  | marker gene DE | [Redefining CpG islands using hidden Markov models](https://academic.oup.com/biostatistics/article/11/3/499/256898?login=false) (MI, MT, AT) | X
| 11.12.2023  | tba | hands-on session #3: single-cell RNA-seq (cell type definition, differential state)  | full scRNA-seq pipeline | [chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data](https://www.nature.com/articles/nmeth.4401) (GP, ER, GB) | X |
| 18.12.2023  | Mark | spatial omics  | spatial statistics | X | X |
 

## Course material

Assuming you have git installed locally, you can check out the entire set of course materials with the following command (from command line):
```
git clone https://github.com/sta426hs2023/material.git
```  
and get updates by running `git pull` at any later time in the same directory.

Alternatively, to retrieve a ZIP file of the repository, you can click on the (green) 'Code' button (top right of main panel) and then click 'Download ZIP'.
