# ATAC-seq Week 2 Exercise Part B


## What is ATAC-seq



## How does ATAC-seq work

## Interesting applications

## Benefits compared to other techniques

| Feature | ATAC-seq | ChIP-seq | RNA-seq |
|---|---|---|---|
| Input material requirement | Lower | Higher | Lower |
| Assay time | Shorter | Longer | Longer |
| Library preparation | In situ | Nuclei isolation required | RNA extraction required |
| Applicability to fresh-frozen tissue | Yes | Yes (with considerations) | Yes |
| Technical difficulty | Easier | More difficult | More difficult |
| What it measures | Chromatin accessibility | Protein-DNA binding | Gene expression |


## Summary
ATAC-seq is an abbreviation for 'Assay for Transposase-Accessible Chromatin with sequencing,' a molecular biology technique employed to evaluate the accessibility of chromatin throughout the entire genome. In this assay, next-generation sequencing (NGS) adapters are loaded onto the transposase, which allows simultaneous fragmentation of chromatin and integration of those adapters into open chromatin regions. The generated library can be sequenced by NGS and thus bioinformatic methods can be used to analyze the regions of the genome with open or accessible chromatin.

The link (technology -> application -> statistics)
| Column 1 | Column 2 | Column 3 |
| -------- | -------- | -------- |
| Data 1   | Data 2   | Data 3   |




## Github Usernames
<table border="0">
<tr><td>

<img src="Pictures/1.png" width="150" alt="Image Description" />


</td><td>

[guidoputignano](https://github.com/guidoputignano) [Guido Putignano]

</td></tr><tr><td>


<img src="Pictures/2.png" width="150" alt="Image Description" />


</td><td>

Jade0904 [Zhihan Zhu]

</td></tr><tr><td>


<img src="Pictures/3.png" width="150" alt="Image Description" />


</td><td>

[NianzhenGu](https://github.com/NianzhenGu) [Nianzhen Gu]

</td></tr><tr><td>


<img src="Pictures/4.png" width="150" alt="Image Description" />


</td><td>

GianfrancoBotta [Gianfranco Botta]

</td></tr></table>
